packages/svelte/src/internal/server/render-context.js

```diff
@@ -28,30 +29,28 @@ export function get_render_context() {
  * @returns {Promise<T>}
  */
 export async function with_render_context(fn) {
-	context = {
+	const store = {
 		hydratable: {
 			lookup: new Map(),
 			comparisons: [],
 			unresolved_promises: new Map()
 		}
 	};
 
-	if (in_webcontainer()) {
+	// Fallback path when AsyncLocalStorage is not available
+	if (in_webcontainer() || als === null) {
```

Comment:
Unfortunately this doesn't work -- `AsyncLocalStorage` is _defined_ in webcontainers, but it just... doesn't actually do anything. It's really dumb.